### ISSUE TITLE
Create Custom Windows VT Sequence

### ISSUE THREAD

**Issue by georgewheatcroft** [microsoft/terminal#8085]
# Description of the new feature/enhancement
As found in issue #1868 (and perhaps any other issues that rely on custom, dynamic changes to terminal appearance based on changes, e.g. ssh to other machine) & discussed on #8064 , there is a need to implement a custom Windows Terminal VT sequence to allow for later work (such as for #1868) to be app-driven. This is beyond the scope of #1868 and hence deserves its own issue ticket

If features such as #1868 are still desired, this would seem like the logical route to go down

**Comment by zadjii-msft** [comment]
Wait hold up, I'm not sure how this is different from #1868 at all. That issue is tracking "author a VT sequence to allow apps to set the tab icon". If I'm not mistaken, this issue is also tracking "author a VT sequence to let the application set the window icon", correct?

**Comment by georgewheatcroft** [comment]
We might be seeing different things (I wouldn't put this past my personal struggle with finding matching issues in github) 

I can't pull up  an issue which matches what you are saying above with regards to "author a VT sequence to allow apps to set the tab icon" 

![image](https://user-images.githubusercontent.com/58346897/97898692-1bfaf780-1d30-11eb-9474-c5693954d7cd.png)

are you referring to just a sub feature noted within #1868 ? 

I might be misunderstanding how things are organised here; this ticket was to start a conversation on carrying out the VT work (which seemingly needs to happen before #1868 can be resolved, as that issue has "gone cold")  because a) this is a feature in itself, which I would see as being beyond the scope of #1868 and b) I could not see any other issue which was specifically working on this, but it sounds like perhaps you can

**Comment by zadjii-msft** [comment]
Hey sorry, I didn't mean that those were literally the titles of the issues, more just what the _gist_ of the issue was fundamentally about.

I think at the end of the day, #1868 needs a new VT sequence to be able to set the tab icon, and part of the work for #1868 is coming up with that sequence. I don't think we really need another thread to track _how_ we're gonna author that sequence, since coming up with the sequence is like 90% of the work of #1868. Yea, that issue's fairly cold, but that's just because there's a lot of work that's being tracked and we can't keep every thread moving simultaneously. We'll come back to it at some point, which is why it's still open ☺️ 

Sorry for the miscommunication!

/dup #1868
